### ISSUE TITLE
Your kindle requires a software update to view this content? CAN'T DOWNLOAD ANYTHING

### ISSUE THREAD

**Issue by xmicro** [apprenticeharper/DeDRM_tools#889]
So I downgraded to 1.17 but now I can't download/view my book (I deleted the old files). What do I do?

**Comment by ElleKayEm** [comment]
Sounds like it's a book that's only available in the KFX format.  You can try a newer Kindle for PC/Mac version 1.19 through 1.24.  If that doesn't work, you won't be able to remove the DRM.

**Comment by xmicro** [comment]
So how do I know if when I buy a book it’s only available in that format without having to buy it? Is there any other way to save it?

**Comment by xmicro** [comment]
> Sounds like it's a book that's only available in the KFX format. You can try a newer Kindle for PC/Mac version 1.19 through 1.24. If that doesn't work, you won't be able to remove the DRM.

Yeah it says I have to update to the latest version of Kindle for PC to download it, and apparently there's a new DRM for KFX in 1.25+ (this includes the latest 1.27). Are there any alternate methods? Thanks.

**Comment by ZolaLa9** [comment]
> So how do I know if when I buy a book it’s only available in that format without having to buy it?

Look under Product Details. If it says "Enhanced Typesetting: Enabled" it's KFX. If it doesn't have Enhanced Typesetting, it will be KF8 or old mobi format. You can also download a free sample to see what format it's in. 

> ... it says I have to update to the latest version of Kindle for PC to download it, and apparently there's a new DRM for KFX in 1.25+ (this includes the latest 1.27). Are there any alternate methods? 

If you're on PC, try 1.24.3. If you're on Mac, try 1.23.1 

See Method 4 here: https://www.mobileread.com/forums/showthread.php?s=d9e1c4f6d94bd169c6da4357f2593de8&t=283371

> Edit: Maybe buying a used Kindle and ripping it from there? Or will it still be un-DRMable?

If the book is KFX-only and the used Kindle is too old for KFX, you won't be able to download it. If the used Kindle has firmware 5.10.2 or later, you won't be able to DeDRM it.

**Comment by xmicro** [comment]
> > So how do I know if when I buy a book it’s only available in that format without having to buy it?
> 
> Look under Product Details. If it says "Enhanced Typesetting: Enabled" it's KFX. If it doesn't have Enhanced Typesetting, it will be KF8 or old mobi format. You can also download a free sample to see what format it's in.
> 
> > ... it says I have to update to the latest version of Kindle for PC to download it, and apparently there's a new DRM for KFX in 1.25+ (this includes the latest 1.27). Are there any alternate methods?
> 
> If you're on PC, try 1.24.3. If you're on Mac, try 1.23.1
> 
> See Method 4 here: https://www.mobileread.com/forums/showthread.php?s=d9e1c4f6d94bd169c6da4357f2593de8&t=283371
> 
> > Edit: Maybe buying a used Kindle and ripping it from there? Or will it still be un-DRMable?
> 
> If the book is KFX-only and the used Kindle is too old for KFX, you won't be able to download it. If the used Kindle has firmware 5.10.2 or later, you won't be able to DeDRM it.

Even if said book said it supports Kindle Fire 1st gen, say, it won't work? So I guess I'm out of luck until the new DRM is cracked I suppose.

**Comment by ElleKayEm** [comment]
It won't matter what generation Fire it supports.  DeDRM has never worked with books from Fires.

**Comment by xmicro** [comment]
> It won't matter what generation Fire it supports. DeDRM has never worked with books from Fires.

Do we know how long before the new DRM is cracked?

**Comment by ElleKayEm** [comment]
No, this is an all volunteer operation.

**Comment by milahu** [comment]
stupid question:
why not update to latest kindle and use screenshots and OCR to rip the book?

similar to [video capture to defeat DRM](https://www.reddit.com/r/Piracy/comments/zueflz/hdmi_capture_to_bypass_drm/)

> Amazon has made a change to their service so the books published since Jan 3 2023 can no longer be downloaded using Kindle for PC or Mac versions prior to 1.39. (Details in [this thread](https://www.mobileread.com/forums/showthread.php?t=351285).) -- [mobileread](https://www.mobileread.com/forums/showthread.php?t=283371)

**Comment by ElleKayEm** [comment]
Sure, as a last resort.  For now, there are still workarounds for most books.  Even books published after Jan 3 2023.  And there's buying elsewhere for many books.
